client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -1230,6 +1242,147 @@ public void testDeleteCalendarEvent() throws IOException {
         assertThat(remainingIds, not(hasItem(deletedEvent)));
     }
 
+    public void testPutDataFrameAnalyticsConfig() throws Exception {
```

Comment:
I'm surprised we don't need any test teardown in the `@After    public void cleanUp()` method. Multiple tests create a config with the id 'test-config'

Reply:
Done, good catch.
I don't know why I didn't hit me, maybe I was lucky with test execution order on my machine?